src/webgpu/web_platform/util.ts

```diff
@@ -386,6 +386,7 @@ export function startPlayingAndWaitForVideo(
           try {
             await callback();
             resolve();
+            video.src = '';
```

Comment:
May I suggest adding `video.srcObject = null;` so that when we're using MediaStream as well, we're also handling it properly>

Reply:
Thanks @beaufortfrancois ! I've uploaded a new commit to address this. But for curious, we use MediaStream through `video.captureStream()`. Is this API set `srcObject` or clean the `srcObject` helps to release the MediaStream?